website/static/css/custom.css

```diff
@@ -1,4 +1,215 @@
-/* your custom css */
+/*--Clear--*/
+* {
+  font-family: 'Poppins', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
+    Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji',
+    'Segoe UI Symbol';
+}
+
+*,
+*:before,
+*:after {
+  margin: 0;
+  box-sizing: border-box;
+}
+body {
+  height: 100%;
+  width: 100%;
+  -webkit-font-smoothing: antialiased;
+}
+
+/*--Grid--*/
+.col-1,
+.col-2,
+.col-3,
+.col-4,
+.col-5,
+.col-6,
+.col-7,
+.col-8,
+.col-9,
+.col-10,
+.col-11,
+.col-12 {
+  padding: 0 20px;
+}
+.col-1 {
+  width: 8.333333%;
+}
+.col-2 {
+  width: 16.666667%;
+}
+.col-3 {
+  width: 25%;
+}
+.col-4 {
+  width: 33.333333%;
+}
+.col-5 {
+  width: 41.666667%;
+}
+.col-6 {
+  width: 50%;
+}
+.col-7 {
+  width: 58.333333%;
+}
+.col-8 {
+  width: 66.666667%;
+}
+.col-9 {
+  width: 75%;
+}
+.col-10 {
+  width: 83.333333%;
+}
+.col-11 {
+  width: 91.666667%;
+}
+.col-12 {
+  width: 100%;
+}
+@media only screen and (max-width: 500px) {
+  .mobile-col-1,
+  .mobile-col-2,
+  .mobile-col-3,
+  .mobile-col-4,
+  .mobile-col-5,
+  .mobile-col-6,
+  .mobile-col-7,
+  .mobile-col-8,
+  .mobile-col-9,
+  .mobile-col-10,
+  .mobile-col-11,
+  .mobile-col-12 {
+    padding: 0 20px;
+  }
+  .mobile-col-1 {
+    width: 8.333333%;
+  }
+  .mobile-col-2 {
+    width: 16.666667%;
+  }
+  .mobile-col-3 {
+    width: 25%;
+  }
+  .mobile-col-4 {
+    width: 33.333333%;
+  }
+  .mobile-col-5 {
+    width: 41.666667%;
+  }
+  .mobile-col-6 {
+    width: 50%;
+  }
+  .mobile-col-7 {
+    width: 58.333333%;
+  }
+  .mobile-col-8 {
+    width: 66.666667%;
+  }
+  .mobile-col-9 {
+    width: 75%;
+  }
+  .mobile-col-10 {
+    width: 83.333333%;
+  }
+  .mobile-col-11 {
+    width: 91.666667%;
+  }
+  .mobile-col-12 {
+    width: 100%;
+  }
+}
+
+/* CSS Variables */
+:root {
+  --ppl-100: #0f0b33;
+  --pl-10: #fafaff;
+  --primary-text: #0f0b33;
```

Comment:
CSS3 yo, FYI browser support: https://caniuse.com/#feat=css-variables

Reply:
namely not supported in IE, classic

Reply:
Opinion: Should I push forward with these variables, or just hard-code em in since it's still an experimental CSS feature?

It's easy for me to swap over.

Reply:
Push forward with em, we can worry about all this at the end.

i.e. do whatever's easiest for you, I'm all for best practices